letsencrypt-compatibility-test/letsencrypt_compatibility_test/util.py

```diff
@@ -25,6 +25,7 @@
 def create_le_config(parent_dir):
     """Sets up LE dirs in parent_dir and returns the config dict"""
     config = copy.deepcopy(constants.CLI_DEFAULTS)
+    config["strict_permissions"] = False
```

Comment:
The `reverter` expects there to be a `strict_permissions` attribute of the configuration. This attribute isn't set in `CLI_DEFAULTS`, so I create it here because normal command line processing is not done in these tests. Perhaps we should just add `strict_permissions=False` to `constants.py` to avoid this problem coming up again (see #803).